### ISSUE TITLE
Versions (0.4, 0.5, 0.6) crashing on repeated CRAZY sudden movement

### ISSUE THREAD

**Issue by Dravic** [Nukem9/dlssg-to-fsr3#145]
Reproduction:

Tested in Witcher 3.

I hopped on Roach, left Novigrad so I am in a reasonably GPU limited area, and just spam turning my camera around. Just keep spinning it left and right with big movements. The game crashes.

I repeated this more than once, so it seems like it's not a fluke. Maybe others can test this as well so we can get to the bottom of it.

Theory: something about the sudden movements causes some sort of overflow when comparing frames that are too different and that crashes the game, this might not be true, just a theory.

Problem: some situations in different games could potentially RARELY have similar outcome and crash your game, so if this isn't just a problem on my end, it might be worth investigating (and fixing if possible).

TL;DR Let me know if it's just me: if you keep spamming the camera movement spinning it left to right then right to left, again and again, until you feel dizzy and then spin it some more. Does it crash the game for you after a couple seconds, too?

**Comment by Dravic** [comment]
It's EXTREMELY repeatable for me in Witcher 3, takes seconds to trigger a crash if I start spinning my camera with my mouse.

**Comment by babalenong** [comment]
Tried to replicate this on darktide, sadly did not crash there even though the UI is majorly messed up lol

**Comment by Dravic** [comment]
> Tried to replicate this on darktide, sadly did not crash there even though the UI is majorly messed up lol

Thanks for testing Darktide.

Waiting for someone else with access to The Witcher 3 who can try to replicate it there.

**Comment by EricFonsecaLima** [comment]
> > Tried to replicate this on darktide, sadly did not crash there even though the UI is majorly messed up lol
> 
> Thanks for testing Darktide.
> 
> Waiting for someone else with access to The Witcher 3 who can try to replicate it there.

It also happens to me. The game crashes after moving the camera with fast movements using the mouse. But I play it with a joystick, so this crashing will not happen while I don't use my mouse. Tested with 0.41, 0.50 and 0.60. Gonna test previous versions of the mod. Btw, my TW3 is freezing after some minutes playing, while doing nothing specific, the music keeps playing but everything else is frozen. Idk if someone else is facing this issue too. (W11, 5900x, 3060Ti 546.17)

**Comment by Dravic** [comment]
> > > Tried to replicate this on darktide, sadly did not crash there even though the UI is majorly messed up lol
> > 
> > 
> > Thanks for testing Darktide.
> > Waiting for someone else with access to The Witcher 3 who can try to replicate it there.
> 
> It also happens to me. The game crashes after moving the camera with fast movements using the mouse. But I play it with a joystick, so this crashing will not happen while I don't use my mouse. Tested with 0.41, 0.50 and 0.60. Gonna test previous versions of the mod. Btw, my TW3 is freezing after some minutes playing, while doing nothing specific, the music keeps playing but everything else is frozen. Idk if someone else is facing this issue too. (W11, 5900x, 3060Ti 546.17)

Thank you for chiming in, it's not just me after all!

**Comment by Dravic** [comment]
Appears to be fixed after briefly testing release version 0.8.1!
